ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+ShareActivity.swift

```diff
@@ -98,7 +98,7 @@ extension BrowserViewController {
 
     // Translate Activity
     if let translationState = tab?.translationState, translationState != .unavailable,
-      Preferences.Translate.translateEnabled.value
+      Preferences.Translate.translateEnabled.value != false
```

Comment:
aren't they the same? `translatedEnabled` has a default value.

Reply:
The default value is nil. Meaning the user hasn't seen the on-boarding yet. False when they disable translations. True when enabled.